src/TRestRawTDSToSignalProcess.cxx

```diff
@@ -0,0 +1,162 @@
+/*************************************************************************
+ * This file is part of the REST software framework.                     *
+ *                                                                       *
+ * Copyright (C) 2016 GIFNA/TREX (University of Zaragoza)                *
+ * For more information see http://gifna.unizar.es/trex                  *
+ *                                                                       *
+ * REST is free software: you can redistribute it and/or modify          *
+ * it under the terms of the GNU General Public License as published by  *
+ * the Free Software Foundation, either version 3 of the License, or     *
+ * (at your option) any later version.                                   *
+ *                                                                       *
+ * REST is distributed in the hope that it will be useful,               *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          *
+ * GNU General Public License for more details.                          *
+ *                                                                       *
+ * You should have a copy of the GNU General Public License along with   *
+ * REST in $REST_PATH/LICENSE.                                           *
+ * If not, see http://www.gnu.org/licenses/.                             *
+ * For the list of contributors see $REST_PATH/CREDITS.                  *
+ *************************************************************************/
+
+//////////////////////////////////////////////////////////////////////////
+/// The TRestRawTDSToSignalProcess is a process used to read a binary file
+/// produced by the read binary files produced with TDS (Tektronix oscilloscope) DAQ,
+/// The results are stores following TRestRawSignalEvent format ,and then it 
+/// processing can continue inside with the REST framework libraries.
+///
+/// ### Parameters
+/// Inherits from TRestRawToSignalProcess: 
+/// * **pedScript**: Not used
+/// * **runScript**: Not used
+/// * **electronics**: You need to write here TDS
+/// * **fileFormat**: Not used
+///
+/// ### Examples
+/// Similar to TRestRawToSignalProcess, jus use TDS as electronics parameter
+/// \code
+///   <addProcess type="TRestRawTDSToSignalProcess" name="virtualDAQ" value="ON">
```

Comment:
If other parameters are not required we could just skip them and write the process definition in a simpler form.

```
<addProcess type="TRestRawTDSToSignalProcess" name="tdsAQS" electronics="TDS"/>
```

Reply:
Indeed, you are right, only `electronics` is a required parameter.
Just updated the description.